content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -341,7 +341,7 @@ Now, when a new Pod is created in the current namespace and using the default
 ServiceAccount, the new Pod has its `spec.imagePullSecrets` field set automatically:
 
 ```shell
-kubectl run nginx --image=nginx --restart=Never
+kubectl run nginx --image=DUMMY_SERVER/nginx --restart=Never
```

Comment:
`_` isn't common in hostnames. Can we swap `DUMMY_SERVER` with `image-registry.some.example` or something like that?

Reply:
Updated.